.travis.yml

```diff
@@ -17,6 +17,9 @@ matrix:
     jdk: openjdk7
   - rvm: jruby-head
     jdk: oraclejdk8
+  - rvm: ruby-head
+    env:
+      - RUBYOPT="--enable-frozen-string-literal"
```

Comment:
Why not put the `# frozen_string_literal: true` magic comment a top of every file so that every Ruby 2.3+ user can reap the benefits? I doubt a lot of people can their Ruby app with this RUBYOPT flag since not every gem is ready for this yet.

Reply:
@bdewater I don't think to update all files in gem, without automatic rubocop/lint checking is useful. It would inconsistent file to file.

I wanted to check that gem is ready,  even without force people who uses gem to fix code. I came to this PR exactly, because enabled in all file of a project `# frozen_string_literal: true`. I don't think I fixed all problems. But at least it would be in the matrix.

Reply:
This record is optional. And it is included ruby head tests without any env.